srcpkgs/neard/files/neard/run

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+sv check dbus >/dev/null || exit 1
+[ -f conf ] && . ./conf
```

Comment:
```suggestion
[ -r conf ] && . ./conf
```

Reply:
Any reason this is better? It seems ` -f` and `-r` are mixed in `run` files across the tree. I mean I suppose we do have to be able to read it to source it.

Reply:
That's interesting, all services in base use -r, but it doesn't seem universal outside of base.